wasmer/limits.go

```diff
@@ -24,6 +24,11 @@ func newLimits(pointer *C.wasm_limits_t, ownedBy interface{}) *Limits {
 	return limits
 }
 
+// NewLimits instantiates a new Limits which describes the Memory used.
```

Comment:
```suggestion
// Instantiates a new `Limits` which describes the `Memory` used.
```

Reply:
When writing Godoc, the comment usually starts with the name of the function/struct it describes. 

The backticks are reasonable.